repository-data/application/src/main/resources/hcm-content/content/documents/administration/labels/global/pagination.yaml

```
@@ -60,7 +60,7 @@
     resourcebundle:id: essentials.pagination
     resourcebundle:keys: [results.indication, page.previous, page.next, page.of, page.navigate-to-page,
       page.page, results.result, no-results-message]
-    resourcebundle:messages: [document(s), Previous, Next, of, Navigate to page, Page,
+    resourcebundle:messages: [document(s), previous, next, of, Navigate to page, Page,
```

Hey, in terms of keeping it consistent - can we de-capitalize `Navigate to page` and `Page`?

 ```suggestion
    resourcebundle:messages: [document(s), previous, next, of, navigate to page, page,
```

---

Hey @DukesColleague , These texts are in use in their current state so we should not be making any change to them as this could affect the modules in the site.

This is actually a problem Juan Luis spotted on Monday and we are rolling the changes back